sds_data_manager/constructs/lambda_layer_construct.py

```diff
@@ -45,9 +46,6 @@ def __init__(
             ),
         )
 
-        self.layer = lambda_.LayerVersion(
-            self,
-            id=f"{id}-Layer",
-            code=code_bundle,
-            compatible_runtimes=[lambda_.Runtime.PYTHON_3_12],
+        super().__init__(
+            id=f"{id}-Layer", code=code_bundle, compatible_runtimes=[runtime], **kwargs
```

Comment:
Makes sense on very high level. I remember you mentioning this in last refactor meeting. Good to see it implemented and working.

In the future, if someone wants to do similar thing, were you looking at this [doc](https://docs.aws.amazon.com/cdk/api/v2/python/aws_cdk.aws_lambda/LayerVersion.html) and then setting everything following `scope` as needed and then return this `super().__init__()` with what we would pass in `LayerVersion` construct?

Reply:
Yes, this is basically just setting default values for us and then calling the `LayerVersion.__init__()` with whatever we want to pass it. The benefit is that this can then be passed directly to the other CDK constructs/functions that reequire Layers rather than having to do `my_construct.layer`